src/main/java/net/sf/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -49,24 +49,24 @@
     private final FieldContentParser parser;
 
 
-    public LatexFieldFormatter() {
-        this(true);
+    public LatexFieldFormatter(JabRefPreferences prefs) {
+        this(true, prefs);
     }
 
-    private LatexFieldFormatter(boolean neverFailOnHashes) {
+    private LatexFieldFormatter(boolean neverFailOnHashes, JabRefPreferences prefs) {
```

Comment:
OK, I added `LatexFieldFormatterPreferences` and `FieldContentParserPreferences`. More are clearly needed to cover the complete `logic`, but at least something.